packages/graphql/src/schema-model/relationship/Relationship.ts

```diff
@@ -35,6 +35,7 @@ export class Relationship {
     public readonly target: Entity;
     public readonly direction: RelationshipDirection;
     public readonly isArray: boolean;
+    public readonly isNullable: boolean;
```

Comment:
together with isArray it's replicating the AttributeType at this point (which also supports Object, Union, Interface) - also what if it's an array of non-nullable items? 
Maybe the current `type` property should be renamed to `relationshipType` and actual `type` should be an AttributeType. Thoughts?

Reply:
I agree the AttributeType structure can become handy here! But I think should not be relevant for the scope of this PR and rather be part of some Schema Model improvements PR!

Reply:
yes, I meant it as part of the Schema Model changes, I agree not in the scope of this PR